core-java-modules/core-java-9/src/test/java/com/baeldung/java9/inputstream/outputstream/InputStreamToOutputStreamUnitTest.java

```diff
@@ -0,0 +1,85 @@
+package com.baeldung.java9.inputstream.outputstream;
+
+import static org.apache.commons.lang3.RandomStringUtils.randomAlphabetic;
+import static org.junit.Assert.assertEquals;
+
+import java.io.*;
+
+import org.apache.commons.io.IOUtils;
+import org.junit.Test;
+
+import com.google.common.io.ByteStreams;
+
+public class InputStreamToOutputStreamUnitTest {
+
+    // buffer size used for reading and writing
+    private static final int BUFFER_SIZE = 8192;
```

Comment:
It seems like overkill to make this a constant and use it only once. Also, the 20480 below isn't a constant.